### PR TITLE
Writing new content

### PR DESCRIPTION
**Pull request by sjlu**

### CODE REVIEW

**Comment by sjlu** [comment]
From previous comments: extending content to explain MVVM in more detail

**Comment by sjlu** [comment]
Hey @AnnJohn, just made a larger revision to the README

File: README.md [hunk 1]
```diff
@@ -74,19 +97,57 @@ you need to add a CSS class? You would have edit elements through
...
-You can see that there is no Javascript involved and our HTML logic
-goes right back to the view. The code is much cleaner and a lot
-easier to write.
+You don't necessarily need to know what this code means just yet, but you can 
+see that there is no Javascript involved and our HTML goes right back to 
+the view. The code becomes much cleaner and a lot easier to write.
+
+### The ViewModel
+
+The ViewModel controls and maintains states of your view and data. It serves
```
Comment (jmburges) [inline README.md:116]: Maybe add an example of a ViewModel from the previous stuff?

File: README.md [hunk 2]
```diff
@@ -74,19 +97,57 @@ you need to add a CSS class? You would have edit elements through
...
+
+In our example, the:
+
+* secretary is the _ViewModel_
+* big boss is the _controller_
+* the phone is the _view_
+
+### Binding
+
+Binding is a way of telling the ViewModel to run code based on certain
```
Comment (jmburges) [inline README.md:135]: Also I would include a concrete example here as well. So something like "look at code x, it will change the color of blah based on blah event from the user. We can say that the color is bound to the textfiled" Or something like that. This is all very abstract which makes it a bit hard to follow

File: README.md [hunk 3]
```diff
@@ -44,27 +44,50 @@ where a person has a first name and a last name
...
-// length of the array
-var people = [{...},{...}]
-var html = ''
-for (var i = 0; i < people.length; i++) {
-  var person = people[i]
-  html.append('<div class="person">...</div>')
-}
-// then write it directly to the page
-$('#people').html(html)
+But lets say we we want to display a table with an unknown
```
Comment (jmburges) [inline README.md:42]: Love the feel the pain, then relieve the pain with Angular approach to exaplining

**Comment by jmburges** [comment]
looks really great. Biggest concern is I think it could use some concrete examples. We got an example of a View, but no real example of a controller, model, viewmodel or "binding"

**Comment by sjlu** [comment]
@jmburges yeah, so this is just the intro, the hard concrete stuff I thought would be in later lessons
